### PR TITLE
Migrate to Polymer 0.8

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

**Comment by blasten** [comment]
I signed it!

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!--
+    @license
```
Comment (cdata) [inline demo/index.html]: Please un-indent this block to match polymerlabs/seed-element :pray:

File: demo/index.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        <div class="progress" horizontal center layout style$="{{compute(ratio)}}">
+          <div class="progress-value"><span>{{ratio}}</span>%</div>
+        </div>
+        
+      </template>
+      
+    </dom-module>
+    
+    <script>
+      Polymer({
```
Comment (cdata) [inline demo/index.html:62]: My understanding is that you cannot define elements inline in the document as of 0.8. It works in Chrome, but fails in other places. Can you double check for me if this demo still works in non-Chrome?

File: demo/index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        is: 'x-progressbar',
+        properties: {
+          min: Number,
+          max: Number,
+          value: Number,
+          ratio: Number
+        },
+        compute: function(ratio) {
```
Comment (cdata) [inline demo/index.html]: `compute` is a little vague. Perhaps we could be a little more clear about what is happening and rename it to `computeStyle`?

File: demo/index.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!doctype html>
+<html>
+  <head>
+    <title>iron-range</title>
+    
+    <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/index.html]: As part of the transition, please change all references to `webcomponents.js` to `webcomponents-lite.js`, which is a package that omits ShadowDOM polyfills.

File: demo/index.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline demo/index.html]: Please update the copyright year to 2015 :cake:

File: index.html [hunk 6]
```diff
@@ -1,22 +1,23 @@
 <!doctype html>
 <!--
-Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
-The complete set of authors may be found at http://polymer.github.io/AUTHORS
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+  Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline index.html]: Ditto re: https://github.com/PolymerElements/iron-range/pull/1/files#r29164289

File: index.html [hunk 7]
```diff
@@ -1,22 +1,23 @@
...
+  The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+  Code distributed by Google as part of the polymer project is also
+  subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
 -->
 <html>
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
   <script src="../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline index.html]: Ditto re: https://github.com/PolymerElements/iron-range/pull/1/files#r29164672

File: index.html [hunk 8]
```diff
@@ -1,22 +1,23 @@
...
-  <link rel="import" href="../core-component-page/core-component-page.html">
-
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
 </head>
-<body unresolved>
+<body>
 
-  <core-component-page></core-component-page>
+  <iron-doc-viewer src="iron-range.html"></iron-doc-viewer>
```
Comment (cdata) [inline index.html]: I think that it's safe to use `<iron-component-page>` for this now :construction_worker:

File: iron-range.html [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline iron-range.html]: Please also include `@license` in this block.

File: iron-range.html [hunk 10]
```diff
@@ -37,7 +40,10 @@
      * @type number
```
Comment (cdata) [inline iron-range.html]: These annotations can be removed. Our tools are able to automatically derive them from the `properties` definition now.

File: iron-range.html [hunk 11]
```diff
@@ -73,36 +88,40 @@
...
-    }
-    
-  });
+  },
+  
+  observers: [
+    '_update(value, min, max, step)'
+  ],
+  
+  _calcRatio: function(value) {
```
Comment (cdata) [inline iron-range.html]: Thanks for making all of these private :star2:

File: test/basic.html [hunk 12]
```diff
@@ -16,16 +16,16 @@
   <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline test/basic.html]: Ditto re: https://github.com/PolymerElements/iron-range/pull/1/files#r29164672

File: test/basic.html [hunk 13]
```diff
@@ -16,16 +16,16 @@
...
   <script src="../../web-component-tester/browser.js"></script>
 
-  <link rel="import" href="../core-range.html">
+  <link rel="import" href="../iron-range.html">
 
 </head>
 <body>
 
-  <core-range></core-range>
+  <iron-range></iron-range>
```
Comment (cdata) [inline test/basic.html]: Consider using [`<test-fixture>`](https://github.com/PolymerElements/test-fixture) so that state doesn't leak across tests.

**Comment by cdata** [comment]
Mostly stylistic feedback, otherwise this looks like a worthy port :ship:

File: bower.json [hunk 14]
```diff
@@ -1,8 +1,13 @@
 {
-  "name": "core-range",
+  "name": "iron-range-behavior",
+  "version": "0.8.0",
```
Comment (cdata) [inline bower.json:3]: @blasten looks like this needs to get updated for polymer#v0.9.0-rc.1

**Comment by notwaldorf** [comment]
Oh and could you squish all your commits next time, please? That's what we've been doing for the other elements...:)

**Comment by blasten** [comment]
sure I will. Only for upgrades right?

**Pull request merged**